code/obj/item/ai_modules.dm

```diff
@@ -46,19 +46,19 @@ AI MODULES
 		return "This law does not exist."
 
 
-	proc/install(obj/machinery/computer/aiupload/comp)
+	proc/install(var/obj/machinery/computer/aiupload/comp, user)
```

Comment:
`var/` in proc arguments is unnecessary and I think discouraged? (If not someone should probably add it to the code guidelines.)